gatsby-config.ts

```diff
@@ -0,0 +1,35 @@
+import type { GatsbyConfig } from 'gatsby'
+
+const config: GatsbyConfig = {
+  siteMetadata: {
+    title: `Eatery Chef`,
+    siteUrl: `https://www.yourdomain.tld`,
+  },
+  // More easily incorporate content into your pages through automatic TypeScript type generation and better GraphQL IntelliSense.
+  // If you use VSCode you can also use the GraphQL plugin
+  // Learn more at: https://gatsby.dev/graphql-typegen
```

Comment:
Do we want to leave this comment here?

Reply:
This part was generated automatically with the `gatsby init` but sure I can remove it